en/docs/deploy/monitor/monitor-server-health.md

```diff
@@ -1,37 +1,30 @@
 # Monitor Server Health
 
 The **Carbon Health Check API** can be used to check the health of WSO2 Identity Server. The sections below guide you through using this API.
-
-!!! note
-    
-    This API is only supported for WSO2 Identity Server that 
-    runs on Java 8 or a later version.
     
-
 !!! info "Health Checkers" 
-    There are three health checkers available by default:
+    There are three health checkers available by default.
 
     1.  **Data sources health checker** - This checker goes through the data
         sources that are configured in the
         `          deployment.toml         ` file and checks if the
         active connection count surpasses a healthy percentage limit (e.g.,
         80%) of the maximum allowed connections count. This checker also
-        tests the connection from each data source to see whether the
+        tests the connection from each datasource to see whether the
         connection is successful.
     2.  **Server startup health checker** - This checker uses the
-        ServerAdmin service to check if the server status is RUNNING.
-    3.  **Super tenant user store health checker** - This checker iterates
-        through configured user stores of the super tenant domain and
+        `ServerAdmin` service to check if the server status is RUNNING.
+    3.  **Super tenant userstore health checker** - This checker iterates
```

Comment:
```suggestion
    3.  **Super tenant userstore health checker**: This checker iterates
```

use `:` in all applicable places

Reply:
We need to look for the best practice here.